pom.xml

```
@@ -25,7 +25,7 @@
 		<kafka-models.version>1.0.25</kafka-models.version>
 		<structured-logging.version>1.9.3</structured-logging.version>
 		<commons-lang3.version>3.11</commons-lang3.version>
-		<private.sdk.version>2.0.77</private.sdk.version>
+		<private.sdk.version>unversioned</private.sdk.version>
```

```suggestion
		<private.sdk.version>2.0.78</private.sdk.version>
```

Similar to comment I left on https://github.com/companieshouse/company-appointments.api.ch.gov.uk/pull/15#pullrequestreview-751356335. We can assume the new sdk version.